.github/workflows/bindings.yml

```diff
@@ -0,0 +1,22 @@
+name: Bindings
```

Comment:
can you name this (and filename) something more general? like `l2` or something? eventually the workflow should compile the system and run l2 tests as well